tools/lib/provision.py

```diff
@@ -247,12 +247,26 @@ def main(options):
         run(["sudo", "service", "redis-server", "restart"])
         run(["sudo", "service", "memcached", "restart"])
     if not options.is_production_travis:
-        # These won't be used anyway
-        run(["scripts/setup/configure-rabbitmq"])
-        # Need to set up Django before using is_template_database_current.
+
+        # Need to set up Django before using is_template_database_current and rabbitmq credentials
         os.environ.setdefault("DJANGO_SETTINGS_MODULE", "zproject.settings")
         import django
         django.setup()
+
+        import pika
+        from django.conf import settings
+        # These won't be used anyway
+        # check if rabbitmq is already configured
+        try:
+            credentials = pika.PlainCredentials(settings.RABBITMQ_USERNAME,
+                                                settings.RABBITMQ_PASSWORD)
+            pika.BlockingConnection(pika.ConnectionParameters(settings.RABBITMQ_HOST,
```

Comment:
It worries me a bit to be duplicating these details from `zerver/lib/queue.py` about how we connect -- the details may change over time, and then we'll either have to update them here as well as there or, worse, this check will just silently break -- so we lose the optimization, or so we never configure rabbitmq.

Can we instead just instantiate a `SimpleQueueClient` and see if that breaks? If that doesn't quite work for some reason, I'd prefer to figure out a refactoring to make it work.